plugins/repoquery.py

```diff
@@ -18,8 +18,8 @@
 
 from __future__ import absolute_import
 from __future__ import unicode_literals
-from datetime import datetime
 from dnfpluginscore import _
+from dnf.cli.output import PackageFormatter, QUERY_TAGS
```

Comment:
OK, I see there is a PR for this in DNF,  you should properly have wait for I was included before starting to use it.